src/Microsoft.AspNet.Authentication.Google/GoogleAuthenticationMiddleware.cs

```diff
@@ -33,23 +33,7 @@ public class GoogleAuthenticationMiddleware : OAuthAuthenticationMiddleware<Goog
             [NotNull] IOptions<ExternalAuthenticationOptions> externalOptions,
             [NotNull] IOptions<GoogleAuthenticationOptions> options,
             ConfigureOptions<GoogleAuthenticationOptions> configureOptions = null)
-            : base(next, dataProtectionProvider, loggerFactory, encoder, externalOptions, options, configureOptions)
-        {
-            if (Options.Notifications == null)
-            {
-                Options.Notifications = new GoogleAuthenticationNotifications();
-            }
-
-            if (Options.Scope.Count == 0)
```

Comment:
The default scopes where here on purpose, don't move them to the options constructor.